app/src/main/java/org/jellyfin/mobile/player/ui/PlayerFullscreenHelper.kt

```diff
@@ -0,0 +1,21 @@
+package org.jellyfin.mobile.player.ui
+
+import android.view.Window
+import androidx.core.view.WindowCompat
+import androidx.core.view.WindowInsetsCompat
+import androidx.core.view.WindowInsetsControllerCompat
+
+class PlayerFullscreenHelper(private val window: Window) {
```

Comment:
Would it make sense to use extension methods on `Window` instead?

Reply:
I want to cache the reference to the controller in the helper.